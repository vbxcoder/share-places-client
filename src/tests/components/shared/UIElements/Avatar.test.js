import React from 'react'
import { shallow } from 'enzyme'

import Avatar from '../../../../shared/components/UIElements/Avatar'

test('Should render Avatar correctly', () => {

  const props = {
    className: 'test-className',
    image: 'test-img-src',
    alt: 'test-img-alt',
    width: 'test-width',
    style: 'test-style'
  }

  const wrapper = shallow(<Avatar {...props} />)
  expect(wrapper).toMatchSnapshot()
})