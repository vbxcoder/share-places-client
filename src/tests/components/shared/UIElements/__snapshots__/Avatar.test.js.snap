// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Avatar correctly 1`] = `
<div
  className="avatar test-className"
  style="test-style"
>
  <img
    alt="test-img-alt"
    src="test-img-src"
    style={
      Object {
        "height": "test-width",
        "width": "test-width",
      }
    }
  />
</div>
`;
